components/Globals/FormField.tsx

```diff
@@ -3,13 +3,27 @@ interface FormProps {
   label: string;
   textarea?: boolean;
   type?: string;
+  notRequired?: boolean;
 }
 
-export default function FormField({ name, label, textarea }: FormProps) {
+export default function FormField({
+  name,
+  label,
+  textarea,
+  type,
+  notRequired,
+}: FormProps) {
+  notRequired = notRequired ? true : false;
```

Comment:
It's a little weird to pass in `notRequired` instead of `required`, like a normal form field. You also wouldn't need this ternary anymore